PD-1470 / 25.04 / PD-1470-document-installing-nvidia-drivers (#3091)
truenas/documentation: content/TruenasApps/_index.md

Restart the system then return to the **Resources Configuration** section of the application to see if expected devices are available.
{{< /expand >}}

## Installing NVIDIA Drivers

> Beginning in TrueNAS 24.10, NVIDIA drivers are no longer automatically installed. Users must instead manually install drivers from the TrueNAS UI.

Reviewer: How about this instead
```suggestion
Beginning in TrueNAS 24.10, NVIDIA drivers are no longer automatically installed. Users must manually install drivers from the TrueNAS UI.
```

> 

Reviewer: When writing a procedure, start steps with the verb action. Remember parallel construction principles apply and procedures are about the action. You can add clarification after the action or just add the result of the action. See the suggested changes below.

> 1. If running TrueNAS version 24.10 or higher, go to **Apps > Installed** and locate the **Configuration** drop-down menu.
>    
> 2. Once you have selected the drop-down menu, click on the **Settings** option to open the **settings configuration panel.**
>    
> 3. Users with compatible GPUs now have an **Install NVIDIA Drivers** checkbox available within the configuration panel.
>    
> 4. To complete installation, check the **Install NVIDIA Drivers** checkbox, and click **Save.**

Reviewer: ```suggestion
If running TrueNAS 24.10 or higher:
1. Go to **Apps > Installed** and click on the **Configuration**.
   
2. Click on **Settings** to open the **settings configuration panel.**
   
3. Select **Install NVIDIA Drivers**, which is available to users with compatible GPUs.
   
4. Select **Install NVIDIA Drivers**, and click **Save.**
```

> {{< trueimage src="/images/SCALE/Apps/AppsInstallNvidiaDrivers.png" alt="Install NVIDIA Drivers" id="Install NVIDIA Drivers" >}}

Reviewer: You can move this image up to the line between step 3 and 4 for better context.
